examples/basic-tracer-node/package.json

```diff
@@ -24,9 +24,9 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.14.0",
-    "@opentelemetry/exporter-jaeger": "^0.14.0",
-    "@opentelemetry/tracing": "^0.14.0"
+    "@opentelemetry/api": "^0.15.0",
```

Comment:
package version should be updated (GitHub doesn't allow to comment on correct line...)